### DISCUSSION TITLE
Editing heading text

### DISCUSSION THREAD

**Discussion by b9Joker108** [GitbookIO/community#528] [0. General bugs and questions]
![Screenshot_20240317_191018_Samsung Internet](https://github.com/GitbookIO/community/assets/147242971/31d607b9-a957-4c51-b37f-93739b1317d7)

Please refer attached screenshot. I have a heading in my document that I wish to edit, but it is persistently difficult to do so. The whole heading is easy to select, and when you do so, a hash is evident in the vertical centre of the heading, to its immediate right. I have tried ever way I can think to engage with the hash with my touchscreen, without success. It is easy to edit at the beginning of the heading, or at the end of the heading, but editing anywhere in the heading's body, is impossible. I have tried every which way to edit within the body of the heading, to no avail. The only workaround, is selecting a word I don't wish to delete and delete it, then type what I wish to edit. This is cumbersome, inefficient, non-standard and counter-intuitive, let alone annoying.

**Comment by aikoii** [comment]
Hey @b9Joker108! From your screenshot, it looks like you're trying to edit a heading on a mobile or tablet? As Addison noted in this [discussion](https://github.com/orgs/GitbookIO/discussions/241#discussioncomment-6249338), GitBook isn't really optimized for editing on mobile or smaller screens. I'm going to close this out in favour of the original post. Thank you!
